src/ext/NetFxExtension/wixlib/NetFx462.wxs

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information. -->
+
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+  <!--
+        .NET Framework installation state properties
+
+        Official documentation can be found at the following location:
+
+           .NET Framework 4.5/4.5.1/4.5.2/4.6/4.6.1/4.6.2 - http://msdn.microsoft.com/en-us/library/w0x726c2(v=vs.110).aspx
+    -->
+
+  <?define NetFx462MinRelease = 394802 ?>
+  <?define NetFx462WebLink = http://go.microsoft.com/fwlink/?LinkId=780597 ?>
+  <?define NetFx462RedistLink = http://go.microsoft.com/fwlink/?LinkId=780601 ?>
```

Comment:
I know how to get the direct download link. My concern was that it might not be a _permanent_ link. The 4.6.1 configuration used `fwlink` which I think is generally permanent.